python-sdk/src/astro/sql/operators/dataframe.py

```diff
@@ -133,6 +135,7 @@ def __init__(
             self.output_table = None
         self.op_args = self.kwargs.get("op_args", ())  # type: ignore
         self.columns_names_capitalization = columns_names_capitalization
+        self.if_exists = if_exists if if_exists else "replace"
```

Comment:
```suggestion
        self.if_exists = if_exists
```

Reply:
Will it ever return `None`?

Reply:
Wouldn't it break if the user doesn't specify `if_exists` for users in the older version of Astro-SDK? that's the reason I kept it optional

Reply:
Can you give an example of that please?